docs/dev-guide/contributing/bugs.rst

```diff
@@ -148,26 +150,32 @@ Pulp holds bug triage as an IRC meeting on Tuesdays and Fridays at 10:30 ET (eit
 A quorum of at least 2 core developers is required to hold triage. Quorum must be established at
 the beginning of the meeting. Developers forming the quorum must stay for the entire duration of
 triage or the meeting must be suspended. Triage will be led by the "triage lead" which is a role
-that rotates on the 1st of each month. The "triage lead" is responsible for reaching out to other
+that can be rotated each month. The "triage lead" is responsible for reaching out to other
 developers to ensure a quorum can be established.
 
+The triage lead uses IRC bot to facilite the meeting. The available set of commands with description
+is provided (e.g. !start, !next, !propose, !accept, !end, etc).
+
 The triage lead will do the following:
 
-#. Announce the meeting in #pulp and #pulp-dev 5 minutes prior to beginning
-#. Start the meeting by confirming there is a quorum (of which they are 1 person of).
-#. Post the link to all `Un-Triaged Bugs`_.
-#. For each issue to be triaged, put the URL of the issue being discussed in the chat and
-   facilitate an agreement on the priority and severity from anyone in the chat. In cases where
-   there is not much feedback, at a minimum the triage lead needs an ack from the other quorum
-   member before moving on. If agreement cannot be reached within 1-2 minutes, skip the bug and let
-   interested parties post their thoughts on the bug.
+#. Announce the meeting in #pulp-dev 10 minutes prior to beginning.
+#. Post the link to all `Un-Triaged Bugs`_ along with the announce of the meeting.
+#. Start the meeting when there is a quorum (of which they are 1 person of, if the triage lead is
```

Comment:
Non-blocking suggestion: Thanks for including this. I think mentioning that the triage leader is part of the quorum should go with the rest of the quorum description, around line 151.